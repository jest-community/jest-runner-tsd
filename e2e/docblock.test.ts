import {expect, test} from '@jest/globals';
import runJest from './runJest';

test('reads `@type` comment in docblock', async () => {
  expect(await runJest('docblock')).toMatchSnapshot();
});
