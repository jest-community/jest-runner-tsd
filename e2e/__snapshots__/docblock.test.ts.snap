// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reads \`@type\` comment in docblock 1`] = `
"PASS ./concat.test.ts
  âœ“ tsd typecheck
Test Suites: 1 passed, 1 total
Tests:       2 passed, 2 total
Snapshots:   0 total
Time:
Ran all test suites.
"
`;
